README.md

```diff
@@ -44,4 +44,5 @@ so one level rendering is not going to be enough.
 ### Snapshot
 
 - use `mount` for small components.
-- use `shallow` like `App` or `*Page` containers or don't use `snapshot` at all.
\ No newline at end of file
+- use `shallow` like `App` or `*Page` containers or don't use `snapshot` at all.
+  - do not wrap component with `<Provider>` but pass `store` as props and `.dive()`
```

Comment:
Also only prop passing checks can use shallow rendering, but the keys always aware of generated snapshot,  be sure children props are passed on the snapshot.